scss/_patterns_code-snippet.scss

```diff
@@ -10,6 +10,10 @@ $color-snippet-heading-bg: rgba($color-x-dark, 0.08);
     .p-code-snippet__block--icon,
     .p-code-snippet__block--numbered {
       margin: 0;
+
+      &.is-wrapped {
```

Comment:
I wonder if we should let it be applied on just `<pre>`, so it can be used on base element, without the whole code snippet component structure (for example when code blocks are generated, and wrapping class is added by JS).

But maybe it's something we can do later if needed.